### PR TITLE
Initial app "repos" and more project structure

### PR DESCRIPTION
**Pull request by txels**
Besides the app, I've started some structure for settings: `base`, `dev`, later on we will have `live` as well, and we can have our own as `dev_carles` and `dev_alex` e.g..

### CODE REVIEW

File: ployst/repos/models.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
+from django.db import models
+from south.modelsinspector import add_introspection_rules
+
+
+class Revision(models.CharField):
```
Comment (txels) [inline ployst/repos/models.py:5]: I've added this custom field not just for saving us from typing the `max_length=40` but rather to later be able to attach behaviour to it.
I was considering making `Revision` a model but I don't think we will actually need that.

**Comment by alexcouper** [comment]
in

File: ployst/repos/models.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    If required in the future, this can be extended to support other
+    repo types, such as mercurial, subversion etc.
+
+    """
+    name = models.CharField(max_length=100)
+    url = models.URLField()
+    active = models.BooleanField(default=True)
+
+    class Meta:
+        verbose_name_plural = 'repositories'
```
Comment (alexcouper) [inline ployst/repos/models.py:34]: Very diligent of you ;)

**Comment by alexcouper** [comment]
Looks good to me!

**Pull request merged**